cli/cmd/report_rules.go

```diff
@@ -0,0 +1,400 @@
+
+	"github.com/lacework/go-sdk/api"
+)
+
+var (
+	// report-rules command is used to manage lacework report rules
+	reportRulesCommand = &cobra.Command{
+		Use:     "report-rule",
+		Aliases: []string{"report-rules", "rr"},
+		Short:   "Manage report rules",
+		Long: `Manage report rules to route reports to one or more email alert channels.		
+
+A report rule has four parts:
+
+  1. Email alert channel(s) that should receive the report
+  2. One or more severities to include
+  3. Resource group(s) containing the subset of your environment to consider
+  4. Notification types containing which report information to send
+`,
+	}
+
+	// list command is used to list all lacework report rules
+	reportRulesListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all report rules",
+		Long:    "List all report rules configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			reportRules, err := cli.LwApi.V2.ReportRules.List()
+			if err != nil {
+				return errors.Wrap(err, "unable to get report rules")
+			}
+			if len(reportRules.Data) == 0 {
+				msg := `There are no report rules configured in your account.
+
+Get started by configuring your report rules using the command:
+
+    lacework report-rule create
+
+If you prefer to configure report rules via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Report Rules.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(reportRules)
+			}
+
+			var rows [][]string
+			for _, rule := range reportRules.Data {
+				rows = append(rows, []string{rule.Guid, rule.Filter.Name, rule.Filter.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework report rule by guid
+	reportRulesShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Show a report rule by id",
+		Long:  "Show a single report rule by it's id.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			var response api.ReportRuleResponse
+			err := cli.LwApi.V2.ReportRules.Get(args[0], &response)
+			if err != nil {
+				return errors.Wrap(err, "unable to get report rule")
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(response)
+			}
+
+			reportRule := response.Data
+			headers := [][]string{
+				[]string{reportRule.Guid, reportRule.Filter.Name, reportRule.Filter.Status()},
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, headers))
+			cli.OutputHuman("\n")
+			cli.OutputHuman(buildReportRuleDetailsTable(reportRule))
+
+			return nil
+		},
+	}
+
+	// delete command is used to remove a lacework report rule by id
+	reportRulesDeleteCommand = &cobra.Command{
+		Use:   "delete",
+		Short: "Delete a report rule",
```

Comment:
Do we want to introduce a `del` alias as it's short for delete.

Reply:
I thought about `rm` once 🤔 - not sure about `del` 

Let us talk about this in standup, if we all agree, we could do it, but if we do, we need to add this alias for ALL `delete` commands.

Reply:
If we'd like to add more aliases we should create a ticket to standardize across commands. We use 'bash' syntactical commands elsewhere like 'ls' for aliases so 'rm' would fit this pattern

Reply:
That's a good point, I like rm.